src/common/lang.js

```diff
@@ -50,3 +50,16 @@ export const load = () => {
 
     addUiLang();
 };
+
+export const showRelatedBanner = () => {
```

Comment:
Could we name this better? What does `related banner` mean?

Reply:
Since the method is in the 'lang.js` I believe it's obvious that what does that mean, don't you think so?

Reply:
So why not just use `showBanner`? What happens if we're going to have other banners here as well?
IMO it's not descriptive enough.